### PR TITLE
Cleanup controls and use Icons instead of buttons

### LINKED ISSUE #3: Controls needs to be more organized
Controls are lying around and doesn't look neat. These can be seperated out as a component and could be shown as an accordion of some kind.

### PR DESCRIPTION
**Pull request by irshadshalu**
Fixes #3  and #4 

Testable version available at http://musicgrid.irshadpi.me

1. Organized controls so that they are consistent
2. Used icons instead of buttons so that UI looks neater
3. Made primary control row sticky so that user can pause/stop while scrolling (This will help when implementing #10 )

When opening:
Screen Shot 2020-06-20 at 12 14 49 AM

After scrolling: 
Screen Shot 2020-06-20 at 12 14 58 AM

### CODE REVIEW

File: src/Controls.svelte [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<script>
+	import { createEventDispatcher, onMount } from 'svelte';
+	import ClipboardJS from 'clipboard';
+
+	const dispatch = createEventDispatcher();
+
+	export let grid;
+	export let config;
```
Comment (wcarss) [inline src/Controls.svelte:8]: this seems surprising to me -- I'm a total novice to svelte, but I'm surprised that config gets declared here and in `App.svelte`

Comment (irshadshalu) [reply src/Controls.svelte:8]: I'm a total noob too. This was my learning exercise. I actually declared it in Control, but the value is getting overwritten by the one from here anyway. I think if we were to use Typescript, I'll feel a bit better about the whole thing

Comment (irshadshalu) [reply src/Controls.svelte:8]: What should be a better alternative?.. any idea how to do this in a clean way in Svelte?

**Comment by wcarss** [comment]
I haven't had a moment yet to pull it down and test it, but it looks alright to me :+1:

**Comment by wcarss** [comment]
Got a chance to test it -- looks good!

it's a little awkward if the user clicks in the first row or two and it beeps then almost immediately beeps again because it started playing, but that doesn't seem like a really big deal.

**Pull request merged**